R/pkg/R/DataFrame.R

```diff
@@ -1146,7 +1147,7 @@ setMethod("head",
 
 #' Return the first row of a SparkDataFrame
 #'
-#' @param x A SparkDataFrame
```

Comment:
Right - they are not - RDD functions are not exported from the packages (not public) and we don't want Rd file generated for them. Please see PR #14626 - we want a separate SetGeneric for non-RDD functions, and then this line documenting both DataFrame and Column parameter can then go to generics.R